### PR TITLE
Integrate omni nav

### PR DESCRIPTION
**Pull request by homeofmatt**

### CODE REVIEW

File: app/assets/javascripts/omni-nav.js [hunk 1]
```diff
@@ -0,0 +1,962 @@
...
+		  gcse.async = true;
+		  gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') +
+		      '//www.google.com/cse/cse.js?cx=' + cx;
+		  var s = document.getElementsByTagName('script')[0];
+		  s.parentNode.insertBefore(gcse, s);
+		})();
+
+	});
+
+})(jQuery);
```
Comment (tatwell) [inline app/assets/javascripts/omni-nav.js]: This is just a copy-paste, yes? If so, I'm going to ignore it.

**Comment by tatwell** [comment]
👍

File: lib/tasks/build.rake [hunk 2]
```diff
@@ -17,15 +17,34 @@ task build: :environment do
...
-    FileUtils.mkdir dist_folder
+    prep_dist
+
     zip rails_asset_path, dist_assets_path
     File.write(dist_cascade_block_path, render(file: 'layouts/cascade-assets.xml', layout: false))
     FileUtils.cp instructions, dist_folder
   end
 end
 
+task build_omni_nav: :environment do
```
Comment (tatwell) [inline lib/tasks/build.rake]: Almost overlooked a couple things:

- Include a description here for `rake -T`. 
- Also, don't forget to update the README to include this and explain why it was created.

**Comment by mirabelrice** [comment]
looks good! 👍

File: README.md [hunk 3]
```diff
@@ -42,6 +42,11 @@ To run a single test:
 
 To deploy changes in this repository to Cascade, please see the [Cascade Accept Testing page](https://kb-smc.chapman.edu/?p=1860) in the Knowledge Base, which includes links to the pages on [Deploying Assets](https://kb-smc.chapman.edu/?p=1894) and [Deploying HTML](https://kb-smc.chapman.edu/?p=1887).
 
+## Deploying the Omni-Nav
+
+After updating the omni-nav, you can specifically compile those assets with the `bin/build_omni_nav` command.
+
+The compiled assets will be located in the `public/_assets` folder. You can deploy them to any website that utilizes the omni-nav from there.
```
Comment (tatwell) [inline README.md]: 👍

**Pull request merged**